doc/content/gateways/lorix-one/lbs.md

```diff
@@ -19,20 +19,39 @@ In the list, select **LoRa Basic Station** and press apply.
 
 {{< figure src="../lorix-one-lora-settings-forwarder-change-bs.png" alt="LORIX One LoRa forwarder Basic Station selection" class="plain" >}}
 
-On the new page, scroll down to the **LoRaWAN Network Server** section and enable it.
+## LNS
+
+Scroll down to the **Configuration** section and enable the **LoRaWAN Network Server**.
 
 {{< figure src="../lorix-one-lora-settings-bs.png" alt="LORIX One LoRa forwarder Basic Station LNS" >}}
 
-Follow the instructions for [Connecting LNS]({{< ref "/gateways/lora-basics-station/lns" >}}).
+To connect the gateway via LNS protocol, follow the instructions for [Connecting LNS]({{< ref "/gateways/lora-basics-station/lns" >}}).
+
+- Configure the **Address** and **Port** with the server address of your deployment and LNS port `8887`
+
+- Enable the **Secured TLS connection** and upload the [LNS Server Certificate]({{< ref "/gateways/lora-basics-station/lns#lns-server-certificate--lns-trust" >}} ) as a **Server authentication certificate**
+
+- Enable the **Client authentication** and copy paste the [LNS Key File]({{< ref "/gateways/lora-basics-station/lns#lns-key-file" >}} )
 
-If using CUPS, following the instructions for [Connecting CUPS]({{< ref "/gateways/lora-basics-station/cups" >}}).
 
 {{< figure src="../lorix-one-lora-settings-bs-lns.png" alt="LORIX One LoRa forwarder Basic Station LNS" class="plain" >}}
 
-Press the **save** button. In the **Control** pane above, press **start** and check the logs in the dedicated pane.
+- Press the **save** button. In the **Control** pane above, press **start** and check the logs in the dedicated pane.
 
 {{< figure src="../lorix-one-lora-settings-bs-control-logs.png" alt="LORIX One LoRa forwarder Basic Station start" >}}
 
+
+## CUPS
+
+Go to the **Configuration** section and enable **Configuration and Update Server**.
+
+{{< figure src="../lorix-one-lora-settings-bs.png" alt="LORIX One LoRa forwarder Basic Station LNS" >}}
+
+To connect the gateway via CUPS protocol, follow the instructions for [Connecting CUPS]({{< ref "/gateways/lora-basics-station/cups" >}}).
+
+Press the **save** button. In the **Control** pane above, press **start** and check the logs in the dedicated pane.
+
+
```

Comment:
For the part below that mentions success messages and errors: I'm seeing `wss:` and port `8887`, which is LBS-related as far as I know, and not CUPS, so it would make more sense to move that part under LBS section